<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMatrix1</class>
 <widget class="QDialog" name="DialogMatrix1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBoxRows">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxColumns">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>561</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="getDataButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>get data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkSymmetryButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>550</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>check symmetry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="detButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>550</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>determinant</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rankButton">
   <property name="geometry">
    <rect>
     <x>492</x>
     <y>550</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>rank</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="transposeTableWidget">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>81</y>
     <width>501</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="transposeButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>transpose</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>29</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Transposed matrix</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fillButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>fill with equal values</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueEdit">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>590</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>590</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter value to fiil matrix with</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scalarMultButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>multiply scalar and matrix</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scalarEdit">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>630</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>630</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter scalar to multiply matrix </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>29</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Rows</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Columns</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
